ci-operator/step-registry/telcov10n/functional/cnf-network/ocp-deploy/telcov10n-functional-cnf-network-ocp-deploy-commands.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+process_inventory() {
```

Comment:
Should this be done in a playbook? 
For better readability and future maintenance.